TestGHACtion/ConsoleApp1/Program.cs

```diff
@@ -1,2 +1,50 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+public class Calculator
+{
+    // Add method
+    public static double Add(double num1, double num2)
+    {
+        return num1 + num2;
+    }
+
+    // Subtract method
+    public static double Subtract(double num1, double num2)
+    {
+        return num1 - num2;
+    }
+
+    // Multiply method
+    public static double Multiply(double num1, double num2)
+    {
+        return num1 * num2;
+    }
+
+    // Divide method
+    public static double Divide(double num1, double num2)
+    {
+        if (num2 == 0)
+        {
+            throw new ArgumentException("Cannot divide by zero.");
+        }
+
+        return num1 / num2;
+    }
+
+    // Main method for testing
+    public static void Main()
+    {
+        // Example usage
+        double resultAdd = Add(5, 3);
+        Console.WriteLine("Addition: " + resultAdd);
+
+        double resultSubtract = Subtract(8, 3);
+        Console.WriteLine("Subtraction: " + resultSubtract);
+
+        double resultMultiply = Multiply(4, 6);
+        Console.WriteLine("Multiplication: " + resultMultiply);
+
+        double resultDivide = Divide(9, 3);
+        Console.WriteLine("Division: " + resultDivide);
```

Comment:
The given code can be refactored in several ways to improve its readability and maintainability. Here's one possible refactoring:

```csharp
﻿using System;

public class Calculator
{
    // Operations Enum
    private enum Operation { Add, Subtract, Multiply, Divide }

    // Perform operation
    private static double PerformOperation(double num1, double num2, Operation operation)
    {
        switch (operation)
        {
            case Operation.Add:
                return num1 + num2;
            case Operation.Subtract:
                return num1 - num2;
            case Operation.Multiply:
                return num1 * num2;
            case Operation.Divide:
                if (num2 == 0)
                {
                    throw new ArgumentException("Cannot divide by zero.");
                }

                return num1 / num2;
            default:
                throw new ArgumentOutOfRangeException("operation", "Invalid operation");
        }
    }

    // Main method for testing
    public static void Main()
    {
        // Example usage
        double resultAdd = PerformOperation(5, 3, Operation.Add);
        Console.WriteLine("Addition: " + resultAdd);

        double resultSubtract = PerformOperation(8, 3, Operation.Subtract);
        Console.WriteLine("Subtraction: " + resultSubtract);

        double resultMultiply = PerformOperation(4, 6, Operation.Multiply);
        Console.WriteLine("Multiplication: " + resultMultiply);

        double resultDivide = PerformOperation(9, 3, Operation.Divide);
        Console.WriteLine("Division: " + resultDivide);
    }
}
```
In this refactored code, we have removed the redundant methods and used a single `PerformOperation` method to handle all four arithmetic operations. We also added an `Operation` enum to make the code more readable and maintainable. Finally, we simplified the exception handling in the `Divide` method by throwing an `ArgumentException`